### ISSUE TITLE
ConfigurationFileHash missing in EventID 16

### ISSUE THREAD

**Issue by Niklas-PDA** [microsoft/SysmonForLinux#181]
**Describe the bug**
We are not able to get ConfigurationFileHash in EventID 16 when changing configuration, only registering as "-"
<Data Name="Configuration">./test.xml</Data><Data Name="ConfigurationFileHash">-</Data>

**To Reproduce**
Change configuration and look at EventID 16

**Sysmon version**
1.3.3 and 1.3.2

**Distro/kernel version**
Red Hat Enterprise Linux 8.9 (Ootpa)
Ubuntu 20.04.6 LTS

**Sysmon configuration**
No configuration avaliable for EventID 16

**Logs**
Jun 25 14:19:02 testhost sysmon[1455]: <Event><System><Provider Name="Linux-Sysmon" Guid="{ff032593-a8d3-4f13-b0d6-01fc615a0f97}"/><EventID>16</EventID><Version>3</Version><Level>4</Level><Task>16</Task><Opcode>0</Opcode><Keywords>0x8000000000000000</Keywords><TimeCreated SystemTime="2024-06-25T12:19:02.807767000Z"/><EventRecordID>88687</EventRecordID><Correlation/><Execution ProcessID="1455" ThreadID="1455"/><Channel>Linux-Sysmon/Operational</Channel><Computer>testhost.domain.local</Computer><Security UserId="0"/></System><EventData><Data Name="UtcTime">2024-06-25 12:19:02.807</Data><Data Name="Configuration">./test.xml</Data><Data Name="ConfigurationFileHash">-</Data></EventData></Event>

**Expected behavior**
Showing example behaviour as seen in windows sysmon:
<Data Name='Configuration'>.\test.xml</Data><Data Name='ConfigurationFileHash'>SHA256=4567546754675467KJHKJ3456LKJ453634KJLH345L6KJH345LK6JH345KJ6H34L</Data>

**Additional context**
n/a

**Comment by hartshorne2** [comment]
Was there any update on this?

**Comment by Niklas-PDA** [comment]
No, we haven't heard anything in any of the issues we have reported...
we have for the moment dropped Sysmon4Linux due to very low activity here and are trying to solve our log needs in other ways.
sad, because sysmon for windows works very well for us...

**Comment by hartshorne2** [comment]
@MarioHewardt are you able to please take a look at this one. Event id 16 is essentially useless if we cannot determine the hash of the updates config file. Currently there is no way to compare the updated config file with the expected config file

**Comment by MarioHewardt** [comment]
Hi - Thanks for reporting this and apologies for the delay. I'm currently fixing an issue in the latest release that prevents Sysmon from running on Debian 11. I expect to release the fix in the next couple of days. Once done, I will take a look at this issue. Once again, apologies for the delay and thanks much for reporting it.

**Comment by MarioHewardt** [comment]
Fixed
